contracts/TokenHolder.sol

```diff
@@ -510,4 +527,15 @@ contract TokenHolder is MultiSigWallet {
             )
         );
     }
+
+    function bytesToBytes4(bytes _input) public pure returns (bytes4 out_) {
+        require(
+            _input.length >= 4,
+            "Input bytes length is less than 4."
```

Comment:
I think it should be _input.length > 4.  We need to parse to bytes 4 if it's more than bytes4.
Also the message should be "Input bytes length is less than or equal to 4".

Reply:
We are going to convert to bytes4. Why should we check that it's bigger then 4 bytes?